tools/webpack/packages.js

```diff
@@ -156,6 +162,14 @@ module.exports = {
 			return `webpack://${ info.namespace }/${ info.resourcePath }`;
 		},
 	},
+	optimization: {
+		splitChunks: {
+			cacheGroups: {
```

Comment:
If this `splitChunks` wasn't there, the `editor` chunk for the Cover block wouldn't be just one file (`block-library/blocks/cover/editor.min.js`), but it would be split into two. Because the Cover block uses the `colord` library, which lives in `node_modules`. By default modules in `node_modules` that are big enough are split into separate, "vendor" chunks. The rationale is better caching, because the vendor libraries don't change so often.

I wanted to avoid this, I wanted to keep the 1:1 relationship between blocks and files. And prevent generating chunks with weird names like `7001.min.js`.

Reply:
> The main question is: does this make the block-library script significantly smaller? The answer is that lazy loading edit functions removes exactly 50% of the code.

It might be one of the reasons why you only noticed 50% of the code moved to chunk as shared code still needs to be bundled in the main entry point. Do you know the number with the shared vendor chunk? Well I still think that 50% saving for the initial load is good enough to roll out this prototype as it will allow us to be less strict about adding more complex functionality to individual blocks 😄